core/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -2596,6 +2596,16 @@ public static SqlQuantifyOperator some(SqlKind comparisonKind) {
     }
   }
 
+  /** Returns the operator for LIKE SOME and LIKE ALL */
+  public static SqlQuantifyOperator like(SqlKind kind, boolean isLike) {
+    SqlKind like = isLike ? SqlKind.LIKE : SqlKind.NOT_LIKE;
+    if (kind == SqlKind.SOME || kind == SqlKind.ALL) {
```

Comment:
nit: Let's convert this to a switch/case with the default case having the AssertionError.